static/js/utils/disaster_event_map_utils.tsx

```diff
@@ -173,12 +174,17 @@ function getGeoJsonFeature(
 ) {
   let geoJson = null;
   if (propVals[geoJsonProp] && !_.isEmpty(propVals[geoJsonProp].vals)) {
-    const geoJsonGeo = JSON.parse(propVals[geoJsonProp].vals[0]);
+    // remove any extra backslashes
```

Comment:
could you add detail here about the source of the extra backslashes?

Reply:
done